Gemfile.lock

```diff
@@ -459,4 +454,4 @@ DEPENDENCIES
   yard
 
 BUNDLED WITH
-   1.17.3
+   1.17.2
```

Comment:
Just confirming that this was intentional? 👀

Reply:
Has no impact would be auto adjusted by weekly automation.